From 34263d1ea99da7b8a680a80601a73149bc9530e5 Mon Sep 17 00:00:00 2001
From: Emekcan Aras <emekcan.aras@arm.com>
Date: Fri, 21 Apr 2023 15:17:21 +0100
Subject: [PATCH] Platform: corstone1000: Increase number of assets

As Corstone1000 stores at boot time few efi variables.
Therefore, number of assets is increased to compansate this early usage.

Upstream-Status: Submitted [https://review.trustedfirmware.org/c/TF-M/trusted-firmware-m/+/20656]
Signed-off-by: Mohamed Omar Asaker <mohamed.omarasaker@arm.com>
Signed-off-by: Emekcan Aras <emekcan.aras@arm.com>
Change-Id: Id8555a09335ce13b80c07a33c4d913f5cb0f9084
---
 platform/ext/target/arm/corstone1000/config_tfm_target.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/platform/ext/target/arm/corstone1000/config_tfm_target.h b/platform/ext/target/arm/corstone1000/config_tfm_target.h
index bf8d2f95f..e96836663 100644
--- a/platform/ext/target/arm/corstone1000/config_tfm_target.h
+++ b/platform/ext/target/arm/corstone1000/config_tfm_target.h
@@ -16,4 +16,12 @@
 #undef PLATFORM_SERVICE_OUTPUT_BUFFER_SIZE
 #define PLATFORM_SERVICE_OUTPUT_BUFFER_SIZE    256

+/* The maximum number of assets to be stored in the Internal Trusted Storage. */
+#undef ITS_NUM_ASSETS
+#define ITS_NUM_ASSETS       20
+
+/* The maximum number of assets to be stored in the Protected Storage area. */
+#undef PS_NUM_ASSETS
+#define PS_NUM_ASSETS        20
+
 #endif /* __CONFIG_TFM_TARGET_H__ */
--
2.17.1

